spec/controllers/api/callbacks/ancestor_ref_controller_spec.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+module Api
+  module Callbacks
+    describe AncestorRefController, type: :controller do
```

Comment:
Should there be a test asserting requests need an auth token to allow these requests from going through? (Is there auth checking?)

Reply:
added auth and associated tests